src/XamlNameReferenceGenerator/XamlNameReferenceGenerator.csproj

```diff
@@ -1,13 +1,22 @@
 <Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
         <TargetFramework>netstandard2.0</TargetFramework>
+        <Product>XamlNameReferenceGenerator ($(TargetFramework))</Product>
```

Comment:
Do we need specific TargetFramework here?

Reply:
Good catch, don't remember why I decided to put it there. Removed.